orderedmap.go

```diff
@@ -60,6 +77,16 @@ func (m *OrderedMap) Len() int {
 	return len(m.kv)
 }
 
+// Max returns the maximum size of the map
+func (m *OrderedMap) Max() int {
```

Comment:
Please use `Capacity` instead. It's more common with the terminology for go slices.